src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,245 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { css } from 'glamor'
+import { onlyS } from '../../theme/mediaQueries'
+import debounce from "lodash.debounce";
```

Comment:
I still think it would be nice to consistently use single quotes and no semicolons in our imports (here and below).

Reply:
Sorry, somehow that change didn't make it in my last commit…